ros_ws/src/control/src/remote_control_node.cpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file remote_control_node.cpp
+ * @brief The main file for the remote control node
+ */
+
+#include "remote_control.h"
+
+const std::string G_TELEOP_TOPIC = "teleop_control";
+
+int main(int argc, char** argv)
+{
+    ros::init(argc, argv, G_TELEOP_TOPIC);
+
+    RemoteControl con;
+
+    ros::Rate loop_rate(30);
+
+    while (ros::ok())
+    {
+        con.update();
+        ros::spinOnce();
+    }
```

Comment:
The variable loop_rate seems to be initialized but never used. Do we need to call loop_rate.sleep() at the end of each iteration of the while loop?

Additionally, the frequency (30) probably needs to be a constant for now, then later it needs to be a parameter (that's a separate task).